src/main/java/at/hannibal2/skyhanni/features/mining/TunnelsMaps.kt

```diff
@@ -180,7 +182,14 @@ object TunnelsMaps {
         }.toMap()
         if (config.autoCommission) {
             clickTranslate.values.firstOrNull()?.let {
+                isCommission = true
                 setActiveAndGoal(it)
+            } ?: run {
+                if (isCommission) {
+                    active = ""
+                    goal = getNext()
```

Comment:
This should be null not getNext() (Since getNext() alwalys returns null in this context)

Reply:
good catch!